### DISCUSSION TITLE
Where did the original project files end up?

### DISCUSSION THREAD

**Discussion by educamundo-dev** [SWivid/F5-TTS#272] [General]
Hi friends!

A while ago I managed to download all the project files that were available here on GitHub. However, when cloning the repository yesterday, I noticed that not all the project files are there.

Could anyone check and help??

Thanks.
![image](https://github.com/user-attachments/assets/596c8eb5-b09a-451f-8ac1-f429633c277b)

**Comment by danielw97** [comment]
Hi,
There was a recent refactor to make the project more compatible as a pip package.
I'm not sure if the original main branch is still available, but if you really need to you could checkout a previous commit.
Hope this helps a bit.
